packs/data/extinction-curse-bestiary.db/starved-staff.json

```diff
@@ -362,7 +533,7 @@
             "alignment": {
                 "value": "NE"
             },
-            "blurb": "",
+            "blurb": "Elite gashadokuro",
```

Comment:
Is this *just* an elite gashadokuro? If so we need to delete this actor.

Reply:
There is enough mechanical variance to warrant its own actor. I added the blurb since it stated it in the statblock. Should I remove that blurb?

Reply:
No, if it's on the statblock then keep it, I just don't want to crack open the book to look at this one. We are finishing book 4 tomorrow.